setup/kubernetes/k3s.md

```diff
@@ -1,97 +1,123 @@
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
 
-> This guide only covers installation on a new Ubuntu 20.04 LTS machine. If you
-> are looking to install on a local machine or an existing host,
-> [Kind](./kind.md) or k3d may be a better choice.
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for both single-node or multi-node clusters. This guide covers the installation
+of K3s onto a new Ubuntu 20.04 LTS machine, but if you want to install Coder on
+a local machine or an existing host, [Kind](./kind.md) or [k3d](https://k3d.io/)
+are better choices.
+
+> This installation method is not officially supported or tested by Coder. If
+> you have questions or run into issues, feel free to reach out using our
+> [community Slack channel](https://cdr.co/join-community).
 
 ## Prerequisites
 
-Before proceeding, please make sure that you have the following:
+Before proceeding, please make sure that:
+
+1. You have an **Ubuntu 20.04 machine**: This can be a bare metal or a virtual
+   machine. For cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr,
+   and DigitalOcean all work well.
 
-1. **Ubuntu 20.04 machine**: This can be a bare metal or virtual machine. For
-   cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr, DigitalOcean
-   work well. Make sure the machine specs satisfies Coder's
-   [resource requirements](../requirements.md)
-1. Network policy exposting ports 22, 80, 443, and 5522 (alternative SSH port)
-1. The following software installed on the machine:
+   Ensure that the machine's specs satisfies
+   Coder's [resource requirements](../requirements.md), since your experience
+   with Coder is dependent on your system specs.
+
+1. You have the following software installed on your machine:
    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
    - [helm](https://helm.sh/docs/intro/install/)
 
-### Resource allocation and performance
+1. Your network policy or firewall accepts incoming traffic on ports 22 (SSH),
+   80 (HTTP), 443 (HTTPS), 5349 (Networking v2 / TURN), and 8443 (Kubernetes
+   API).
 
-Your experience with Coder is dependent on your system specs. See our
-[resource requirements](../requirements.md) for more details.
+## Step 1: Change the default SSH port
 
-## Change the default SSH port
+To allow [SSH into
+workspaces](https://coder.com/docs/coder/v1.20/workspaces/ssh), you must change
+the host's default SSH port to free up port 22. You may also need to modify your
+firewall to accept incoming traffic from the alternative port (e.g 5522).
 
-If you want to allow
-[SSH into workspaces](https://coder.com/docs/coder/v1.20/workspaces/ssh), you
-will need to change the host's default SSH port to free up port 22.
+> If you don't know how to change the SSH port in Linux, please review this
+> [guide from Linuxize](https://linuxize.com/post/how-to-change-ssh-port-in-linux/)
```

Comment:
Should mark this as optional if you're going to be using net v2

Reply:
I went to add brief info about networking v2 for the user, and it appears that we've removed the option from the UI? Should we leave as-is until this feature is out of alpha, then update the doc with a caveat?

Reply:
![Screen Shot 2021-06-28 at 10 53 47 AM](https://user-images.githubusercontent.com/22407953/123657841-2d34cf00-d7ff-11eb-87ce-b058ab5392b7.png)

1.20 has the networking v2 option under Manage > Admin > Infrastructre. In 1.21 this will likely be in the workspace providers section